src/network/networkd-dhcp4.c

```diff
@@ -631,14 +636,24 @@ int dhcp4_configure(Link *link) {
                         return r;
         }
 
-        /* Always acquire the timezone and NTP */
-        r = sd_dhcp_client_set_request_option(link->dhcp_client, SD_DHCP_OPTION_NTP_SERVER);
-        if (r < 0)
-                return r;
+        /* RFC7844 section 3.6.:
+           The client intending to protect its privacy SHOULD only request a
+           minimal number of options in the PRL and SHOULD also randomly shuffle
+           the ordering of option codes in the PRL.  If this random ordering
+           cannot be implemented, the client MAY order the option codes in the
+           PRL by option code number (lowest to highest).
+        */
+        /* Otherwise, always acquire the timezone and NTP */
+        /* NOTE: there should be a configuration variable for these options? */
```

Comment:
maybe we should request this only if dhcp_use_timezone is set. A patch for that would be very welcome

Reply:
I've created #6725 and #6726. I'll remove the ``note`` after we check that those patch are fine.